bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/PowerMaxBinding.java

```diff
@@ -0,0 +1,881 @@
+     */
+    private boolean triggerItemsUpdate = false;
+
+    /** Boolean indicating whether or not connection is established with the alarm system */
+    private boolean connected = false;
+
+    public PowerMaxBinding() {
+    }
+
+    /**
+     * Activates the binding. Actually does nothing, because on activation
+     * OpenHAB always calls updated to indicate that the config is updated
+     * Activation is done there
+     */
+    @Override
+    public void activate() {
+        logger.debug("Activate PowerMax alarm binding");
+    }
+
+    /**
+     * Deactivates the binding
+     */
+    @Override
+    public void deactivate() {
+        logger.debug("Deactivate PowerMax alarm binding");
+        closeConnection();
+    }
+
+    /**
+     * @{inheritDoc
+     */
+    @Override
+    protected long getRefreshInterval() {
+        return 20000; // 20 seconds
+    }
+
+    /**
+     * @{inheritDoc
+     */
+    @Override
+    protected String getName() {
+        return "PowerMax Refresh Service";
+    }
+
+    /**
+     * @{inheritDoc
+     */
+    @Override
+    protected void execute() {
+        logger.debug("PowerMax alarm Execute");
+
+        if (!isProperlyConfigured()) {
+            logger.debug("execute(): not yet properly configured");
+            return;
+        }
+
+        //
+        // Off items linked to motion sensors after the delay defined by
+        // the variable motionOffDelay
+        //
+
+        long now = System.currentTimeMillis();
+        PowerMaxPanelSettings settings = PowerMaxPanelSettings.getThePanelSettings();
+        PowerMaxState updateState = null;
+        if ((currentState != null) && (settings != null)) {
+            for (int i = 1; i <= settings.getNbZones(); i++) {
+                if ((settings.getZoneSettings(i) != null)
+                        && settings.getZoneSettings(i).getSensorType().equalsIgnoreCase("Motion")
+                        && (currentState.isSensorTripped(i) == Boolean.TRUE)
+                        && (currentState.getSensorLastTripped(i) != null)
+                        && ((now - currentState.getSensorLastTripped(i)) > motionOffDelay)) {
+                    if (updateState == null) {
+                        updateState = new PowerMaxState();
+                    }
+                    updateState.setSensorTripped(i, false);
+                }
+            }
+        }
+        if (updateState != null) {
+            updateItemsFronAlarmState(PowerMaxSelectorType.ZONE_STATUS, updateState);
+            currentState.merge(updateState);
+        }
+
+        if (PowerMaxCommDriver.getTheCommDriver() != null) {
+            connected = PowerMaxCommDriver.getTheCommDriver().isConnected();
+        }
+
+        if (connected && (currentState.isPowerlinkMode() != null) && currentState.isPowerlinkMode().equals(Boolean.TRUE)
+                && (currentState.getLastKeepAlive() != null) && ((now - currentState.getLastKeepAlive()) > 60000)) {
```

Comment:
OK
The idea is to check a particular message is received at least once every minute.